os/test/src/PathTests.scala

```diff
@@ -3,17 +3,20 @@ package test.os
 import java.nio.file.Paths
 
 import os._
+import os.Path.{platformPrefix}
 import utest.{assert => _, _}
 import java.net.URI
 object PathTests extends TestSuite {
+  def enableTest = true
```

Comment:
If `enableTest` is `true`, we should be able to remove it and remove all the conditionals guarding the test cases

Reply:
I'll remove them.